axelrod/tests/integration/test_filtering.py

```diff
@@ -1,10 +1,13 @@
+import random
 import unittest
 
-from axelrod import all_strategies, filtered_strategies, seed
+from axelrod import short_run_time_strategies, filtered_strategies, seed
 
 from hypothesis import example, given, settings
 from hypothesis.strategies import integers
 
+strategies = random.sample(short_run_time_strategies, 20)
```

Comment:
We have the [`property.strategy_lists`](https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/property.py#L12) that we can use instead of this.

So we'd have:

```
from axelrod.tests.property import strategy_lists
```

and then use:

```
@given(strategies=strategy_lists(min_size=20, max_size=20))  # or other parameters?
```